src/WebJobs.Script/StorageProvider/IDelegatingHandlerProvider.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Net.Http;
 
-namespace Microsoft.Azure.WebJobs.Script.WebHost.Storage
+namespace Microsoft.Azure.WebJobs.Script
```

Comment:
Are you referring just to the namespace change (set it back to `Microsoft.Azure.WebJobs.Script.WebHost.Storage`) or also the project movement: from `WebJobs.Script.WebHost` to `WebJobs.Script`?

The interface was moved to `WebJobs.Script` as `HostAzureTableStorageProvider` dependency.

Reply:
I missed the move. This type was public before, but the only reasonable consumer would have been Core Tools (wouldn't expect usage there, but worth checking to avoid wasted cycles). Would be great if we could avoid this move/change, but this is something we can live with as this is a type I'm hoping we'll be able to remove down the road.

Reply:
Checked Core Tools and there's no explicit reference to the interface nor the implementation type (which I kept in the original namespace/project). It's registered on DI [here](
https://github.com/Azure/azure-functions-core-tools/blob/ae9eff234b96a6120d9106a739252aa200476c31/src/Azure.Functions.Cli/Actions/HostActions/Startup.cs#L87).

All this said, I've been doing some digging and I think I have good news. When `Webjobs.Host.Storage` [was migrated to Track 2 SDKs](https://github.com/Azure/azure-webjobs-sdk/pull/2764), we stopped using the `DelegatingHandlerProviders` with these clients to limit max number of connections in places where we used them [with the previous SDK](https://github.com/Azure/azure-webjobs-sdk/blob/8c90eb9bf2f6562cdd82463371fa38f427206315/src/Microsoft.Azure.WebJobs.Host.Storage/StorageServiceCollectionExtensions.cs#L83). This would make sense because the new SDKs invested on connection handling and pooling. 

On the Host, the only usage of these `DelegatingHandlers` is with the `CloudTableClient` used in _Scale Metrics_ and that belongs to previous SDK.  I would say that the solution to all our concerns is, as part of this migration, to not use them at all. And even remove the types 😊.